OSDOCS-16301 created z-stream release notes for 4-18-25 (#99815)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

====

// 4.18.25
[id="ocp-4-18-25_{context}"]
=== RHBA-2025:16732 - {product-title} {product-version}.25 bug fix update

Issued: 01 October 2025

{product-title} release {product-version}.25 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:16732[RHBA-2025:16732] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:16729[RHSA-2025:16729] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.18.25 --pullspecs
----

[id="ocp-4-18-25-enhancements_{context}"]
==== Enhancements

* With this update, {product-title} {product-version}.25 is upgraded to Kubernetes 1.31.12, ensuring a secure, stable, and up-to-date platform. This enhancement improves stability, enhances security, and boosts performance by incorporating the latest upstream changes and fixes. (link:https://issues.redhat.com/browse/OCPBUGS-60511[OCPBUGS-60511])

> * With this update, the collection of command line logs from `virt-launcher` pods across a Kubernetes cluster are enabled. JSON-encoded logs are saved at the path `namespaces/<namespace_name>/pods/<pod_name>/virt-launcher.json`, facilitating troubleshooting and debugging of virtual machines. (link:https://issues.redhat.com/browse/OCPBUGS-61656[OCPBUGS-61656])

Reviewer: ```suggestion
* With this update, the collection of command line logs from `virt-launcher` pods across an {product-title} cluster are enabled. JSON-encoded logs are saved in the `namespaces/<namespace_name>/pods/<pod_name>/virt-launcher.json` path, facilitating troubleshooting and debugging of virtual machines. (link:https://issues.redhat.com/browse/OCPBUGS-61656[OCPBUGS-61656])
```
